### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ColorContrast: The drop down color contrasts is below 3:1 #11194

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#3412]
["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:-A11y_AzureDataStudioStayHealthy;-A11yTCS;-A11yWCAG2.1;-Benchmark-DesktopApp;-MAS1.4.11;-SQL Azure Data Studio;-FTP;-ColorContrast;-A11ySev2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows) 
 appears.
4. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu.
Python Dependencies Wizard will appear.
5. Provide any Input file details & Next.
6.  Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
Issue1:
When the color contrast of the dropdown is checked the ratio is 2.847:1

Issue2:
The color contrast of the selected item to the item, not selected  is 1.604:1

**Expected:**
Issue1:
The color contrast of the dropdown should be 3:1 or greater.

Issue2:
The color contrast of the  selected item to the item, not selected should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in viewing the separate sections in the lsit .

**Attachment link for Reference:**
